requirements.txt

```diff
@@ -1,2 +1,4 @@
 Flask==1.0.2
 requests==2.18.4
+flasgger==0.9.1
```

Comment:
perhaps this should be something in dev-requirements and we only try to import it? this just seems like a nice-to-have to generate the docs but seems strange to require it to run the service. idk, that's mostly just my opinion :P